tests/test_cms_menus.py

```diff
@@ -839,6 +836,91 @@ def test_menu_with_softroot_page_rendering(self):
 
         self.assertTreeQuality(soft_root, mock_tree, 'level', 'title')
 
+    def test_menu_with_hidden_softroot_page_rendering(self):
+        """
+        Tree in fixture :
+               root
+                   aaa (soft_root,hide_node)
+                       aaa1
+                           ccc
+                               ddd
+                       aaa2
+                   bbb
+        tag: show_menu 0 100 100 100
+        expected result:
+            renders just the softroot page, with no menu nodes
```

Comment:
```suggestion
            renders just the softroot page, with no menu nodes. It renders no nodes because the soft_root and hide_node are True and the requirement is that when rendering a node that is soft_root and hide_node the navigation should be empty
```